### PR TITLE
Preparation for release 1.2.0

### PR DESCRIPTION
**Pull request by matq007**
- Updated documentation
- Removed `conda` profile
- Updated internal testing

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/rnafusion branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/rnafusion)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -7,34 +7,39 @@ This document describes the output produced by the pipeline.
...
-* [EricScript](#ericscript)
-* [FastQC](#fastqc) - read quality control
-* [FusionCatcher](#fusioncatcher)
-* [FusionInspector](#fusion-inspector)
-* [fusion-report](#fusion-report)
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
-* [Pizzly](#pizzly)
-* [Squid](#squid)
-* [Star-Fusion](#star-fusion)
+- [nf-core/rnafusion: Output](#nf-corernafusion-output)
```
Comment (MaxUlysse) [inline docs/output.md:10]: Can you add `<!-- omit in toc -->` to the header, that way it won't be in the automatic TOC?

File: docs/output.md [hunk 2]
```diff
@@ -7,34 +7,39 @@ This document describes the output produced by the pipeline.
...
+  - [Pipeline overview](#pipeline-overview)
```
Comment (MaxUlysse) [inline docs/output.md:11]: I think we can do the same here

File: download-singularity-img.nf [hunk 3]
```diff
@@ -32,7 +32,7 @@ def helpMessage() {
     Mandatory arguments:
       --outdir [path]               Output directory for downloading
       -profile [str]                Configuration profile to use. Can use multiple (comma separated)
-                                    Available: standard, conda, docker, singularity, awsbatch, test
+                                    Available: standard, docker, singularity, awsbatch, test
```
Comment (MaxUlysse) [inline download-singularity-img.nf:35]: I think we can remove awsbatch as well from the comments, no one is going to use singularity on AWS

File: test.sh [hunk 4]
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 # Test tools only
-nextflow run nf-core/rnafusion -profile munin -c /data1/references/rnafusion/1.1.0/munin-singularity.config --reads "/data1/references/rnafusion/1.1.0/reads_{1,2}.fq.gz" --arriba --star_fusion --fusioncatcher --ericscript --pizzly --squid --debug
+nextflow run nf-core/rnafusion -profile munin --ericscript --reads "/data1/references/rnafusion/1.1.0/reads_{1,2}.fq.gz" --arriba --star_fusion --fusioncatcher --ericscript --pizzly --squid --debug
```
Comment (MaxUlysse) [inline test.sh:4]:
```suggestion
nextflow run nf-core/rnafusion -profile munin --reads "/data1/references/rnafusion/1.1.0/reads_{1,2}.fq.gz" --arriba --star_fusion --fusioncatcher --ericscript --pizzly --squid --debug
```

Comment (MaxUlysse) [reply test.sh:4]: --ericscript was already there

File: docs/output.md (hunk 1 above)
Comment (matq007) [reply docs/output.md]: done

File: docs/output.md (hunk 2 above)
Comment (matq007) [reply docs/output.md]: done

File: docs/output.md [hunk 5]
```diff
@@ -1,40 +1,45 @@
+<!-- omit in toc -->
```
Comment (MaxUlysse) [inline docs/output.md:1]: I usually put it at the end of the line that has the stuff I want to omit, but it does make sense to put it before as well

**Comment by matq007** [comment]
Ah you are right, the link don't work after removal of `--`. My VSCodium plugin removed it... It always updated when I try to add them back, have to disable it 😞

**Pull request merged**